snippets/rust.json

```diff
@@ -416,6 +416,13 @@
         ],
         "description": "impl … { … }"
     },
+    "inline": {
+        "prefix": "inline",
+        "body": [
+            "#[inline]"
```

Comment:
You write `inline` it replaces it with `#[inline]`.
I don't think it worth it, but I'll want other people.
If people want it, I'll merge it.